specification/conformance/conformance_rules/columns/capacityreservationid.json

```diff
@@ -0,0 +1,255 @@
+        "CapacityReservationId-C-002-M",
+        "CapacityReservationId-C-003-C",
+        "CapacityReservationId-C-007-C"
+      ]
+    }
+  },
+  "CapacityReservationId-C-001-M": {
+    "Function": "Type",
+    "Reference": "CapacityReservationId",
+    "EntityType": "Column",
+    "CRVersionIntroduced": "1.2",
+    "Status": "Active",
+    "Notes": "",
+    "ApplicabilityCriteria": [],
+    "Type": "Static",
+    "ValidationCriteria": {
+      "MustSatisfy": "CapacityReservationId MUST be of type String.",
+      "Keyword": "MUST",
+      "Requirement": {
+        "CheckFunction": "TypeString",
+        "ColumnName": "CapacityReservationId"
+      },
+      "Condition": {},
+      "Dependencies": []
+    }
+  },
+  "CapacityReservationId-C-002-M": {
+    "Function": "Format",
+    "Reference": "CapacityReservationId",
+    "EntityType": "Column",
+    "CRVersionIntroduced": "1.2",
+    "Status": "Active",
+    "Notes": "",
+    "ApplicabilityCriteria": [],
+    "Type": "Static",
+    "ValidationCriteria": {
+      "MustSatisfy": "CapacityReservationId MUST conform to StringHandling requirements.",
+      "Keyword": "MUST",
+      "Requirement": {
+        "CheckFunction": "FormatString",
+        "ColumnName": "CapacityReservationId"
+      },
+      "Condition": {},
+      "Dependencies": []
+    }
+  },
+  "CapacityReservationId-C-003-C": {
+    "Function": "Composite",
+    "Reference": "CapacityReservationId",
+    "EntityType": "Column",
+    "CRVersionIntroduced": "1.2",
+    "Status": "Active",
+    "Notes": "",
+    "ApplicabilityCriteria": [],
+    "Type": "Static",
+    "ValidationCriteria": {
+      "MustSatisfy": "CapacityReservationId nullability is defined as follows:",
+      "Keyword": "MUST",
+      "Requirement": {
+        "CheckFunction": "AND",
+        "Items": [
+          {
+            "CheckFunction": "CheckConformanceRule",
+            "ConformanceRuleId": "CapacityReservationId-C-004-C"
+          },
+          {
+            "CheckFunction": "CheckConformanceRule",
+            "ConformanceRuleId": "CapacityReservationId-C-005-C"
+          },
+          {
+            "CheckFunction": "CheckConformanceRule",
+            "ConformanceRuleId": "CapacityReservationId-C-006-C"
+          }
+        ]
+      },
+      "Condition": {},
+      "Dependencies": [
+        "CapacityReservationId-C-004-C",
+        "CapacityReservationId-C-005-C",
+        "CapacityReservationId-C-006-C"
+      ]
+    }
+  },
+  "CapacityReservationId-C-004-C": {
+    "Function": "Nullability",
+    "Reference": "CapacityReservationId",
+    "EntityType": "Column",
+    "CRVersionIntroduced": "1.2",
+    "Status": "Active",
+    "Notes": "",
+    "ApplicabilityCriteria": [],
+    "Type": "Dynamic",
+    "ValidationCriteria": {
+      "MustSatisfy": "CapacityReservationId MUST be null when a charge is not related to a capacity reservation.",
+      "Keyword": "MUST",
+      "Requirement": {},
+      "Condition": {},
```

Comment:
Should we define a condition to support the value `MustSatisfy`?
`MustSatisfy`: "CapacityReservationId MUST be null when a charge is not related to a capacity reservation."

Reply:
Ok, understood.